### DISCUSSION TITLE
Google block fermata auto

### DISCUSSION THREAD

**Discussion by fcaronte** [AndreyPavlenko/Fermata#123] [General]
Seem your app disappear from android auto app list, and I have the same 7.0.614134 version as two day ago when was working correctly. Any feedback?

**Comment by podimonanil** [comment]
Same Here.!!! Try rolling back to previous version of Android Auto

**Comment by fcaronte** [comment]
Ok seem on the 7.0.614124 is working![](https://i.imgur.com/4zOsIEJ.jpg)

**Comment by fcaronte** [comment]
Ok app disappeared again so we can confirm package name is blacklisted by Google on any AA Version

**Comment by patrik-malina** [comment]
Same with me 😢

**Comment by Muro0101** [comment]
Same here...I hate Google 🤬😡

**Comment by Elabrins8** [comment]
Same here.. 😫

**Comment by podimonanil** [comment]
I rolled back to 7.0.164124, first time it was showing in car, when I reconnected it was gone.
Again rolled back to 6.9
Fermata Auto was back again, but when I try to connect a second time, it was gone.
Goggle Black listed Fermata .
Someone please find a walkaround.

**Comment by podimonanil** [comment]
Rolling back to previous versions won't work anymore,
It may show at first, but it will disappear when you try to connect a second time .

**Comment by AndreyPavlenko** [comment]
Temporary solution - https://forum.xda-developers.com/t/fermata-media-player-audio-and-video-player-for-android-auto.4079519/post-85936477

**Reply by Muro01adanali** [reply]
Hi Andre, I just downloaded it. But it doesn't open, I was using Fermata for 6 months without problems, but since a week ago it's giving problems again.

**Comment by fcaronte** [comment]
Installed the renamed version and for now is working

**Comment by Alfredo2190** [comment]
> Parece que su aplicación desaparece de la lista de aplicaciones automáticas de Android, y tengo la misma versión 7.0.614134 que hace dos días cuando funcionaba correctamente. ¿Cualquier retroalimentación?

Me pasa lo mismo, hace dos días dejó de funcionar fermata auto y control... En las aplicaciones a ver en AA no se ve ni se puede agregar. 🤔😪

**Comment by Alfredo2190** [comment]
Fermata auto control 1.8 7 pude instalar. Pero fermata auto arm64 no deja.. Podrás hacer un fermata auto universal común.. Que no sean para base arvm 64????

**Comment by Muro0101** [comment]
> Temporary solution - https://forum.xda-developers.com/t/fermata-media-player-audio-and-video-player-for-android-auto.4079519/post-85936477

Google blocked it again 😭😭😭

**Comment by Timur82** [comment]
Still not working please help

**Comment by fcaronte** [comment]
Is work correctly probably you continue to use the old version. Uninstall it and then use kingInstaller to install the new apk, if you install normally will never work

**Comment by Grafden** [comment]
Ок благодарю уже решил проблему ссылкой выше

**Comment by ccpoon1982** [comment]
same here in Malaysia. tried update to fermata 1.8.6 with AA stay in old version 6.8.613354, first it works fine. but second day, when i stop my car and restart another day, it disappear. will need to clear data / clear cache every day in order to let Fermata to shows again. every day routine....really tiring & really a issue. Hope Sir Andrey can fix this permanently. thanks a lot.

**Comment by fcaronte** [comment]
As write before here, you need the 1.8.7 installed using kingInstaller and you need to uninstall the old version

**Comment by Cesarlgo** [comment]
Hi help me, please. Have a problem with the 1.8.7 version. I have installed this version appear in AA but when I start the app in AA, give me the message "No message has been received ...". Could you help me please. Thank.

**Comment by lapsio** [comment]
> Hi help me, please. Have a problem with the 1.8.7 version. I have installed this version appear in AA but when I start the app in AA, give me the message "No message has been received ...". Could you help me please. Thank.

I have the same issue. I'm on AA 7.0.614134-release, Android 11. App is visible in AA however after launching it gives black screen with "No messages have been received blah blah". I tried both installing it over 1.8.6 as well as uninstalling everything and installing 1.8.7 using kinginstaller. I have enabled installation from untrusted source in AA dev settings.

**Comment by lapsio** [comment]
> Hi help me, please. Have a problem with the 1.8.7 version. I have installed this version appear in AA but when I start the app in AA, give me the message "No message has been received ...". Could you help me please. Thank.

After uninstalling everything several times, dozen of reboots and repeaded installations it finally launched. I think this "No message..." screen is bug that occurs due to some cached garbage in Android Auto. I uninstalled everything, rebooted phone, then installed again 1.8.7 using Kinginstaller, rebooted, launched app on car (without launching it on phone before that) and it just worked.

Unfortunately I'm unable to browse files. Only YouTube and WebBrowser works. Adding folders on phone doesn't seem to propagate to AA app. The same about settings, those two apps seem to be completely independent. Even after allowing app on phone and adding folders on phone, they're still not available on AA.

**Comment by RealAndyhuang** [comment]
Hello everyone, I'm using Android 12, why I can't install the kinginstaller? 
just No response when I press kinginstaller apk.

**Comment by Cesarlgo** [comment]
Hi, Could you tell me where I can download KingInstaller? Please.

**Comment by lapsio** [comment]
> Hi, Could you tell me where I can download KingInstaller? Please.

It's available at the very beginning of xda thread mentioned here by AndreyPavlenko

**Comment by RealAndyhuang** [comment]
> Hi, Could you tell me where I can download KingInstaller? Please.

https://inceptive.ru/blog/king-installer-no-root/

**Comment by Cesarlgo** [comment]
Thank you very much for the information.  I continue with the message problem when starting Fermata in AA.  Has anyone been able to fix it?

**Comment by fcaronte** [comment]
I think you need to remove all Android auto related app, delete Android auto data and do all from scratch

**Comment by gaduaxe** [comment]
hi @fcaronte @AndreyPavlenko 

If I have root phone & patched Fermata using AA AIO Tweaker, does it work with latest Google AA?
And with root phone, is there any others method to tweak/patch Gg Android Auto to have fermata working?

**Comment by fcaronte** [comment]
I have no idea how work with root, I'm using without

**Comment by RealAndyhuang** [comment]
@fcaronte 
I can't install kinginstaller on Android 12, do you know reason?

**Comment by gaduaxe** [comment]
> I have no idea how work with root, I'm using without

I tried root with AIO Tweaker and latest AA & Fermata -> it works

**Comment by fcaronte** [comment]
@RealAndyhuang you make some mistake, I get some video from a friend with a s21 with andorid 12 and was able to install kingInstaller and then fermata without problem. Give a check on your step

**Comment by RealAndyhuang** [comment]
@fcaronte thanks, I already install the kinginstaller then the 1.88.
It can not be install in chrome, but it can be install from file explorer. 
Thanks for helping

**Comment by fcaronte** [comment]
@RealAndyhuang probably you miss to add needed permission to Chrome but if you just fix is okay with your way too 💪🏻

**Comment by Muro0101** [comment]
It's working two days in a row now...

**Comment by Alfredo2190** [comment]
Les cuento. Tengo Samsung 7 rooteado y funciona bárbaro el Fermata Auto 1.8.8 ...Y el Fermata control bien 👍
Intento instalar el mismo Fermata 1.8.8 en un Nokia 2.4 no rooteado y no sólo no lo pude instalar sino que tampoco me permite instalar KingInstaller .. para instalar kinginstaller me pide que sea root.

**Comment by demostyler** [comment]
Blocked again :( europe

**Comment by Alfredo2190** [comment]
Buen día.Había comentado más arriba que no funcionaba en móvil sin root (nokia2.4) . Bueno, hace una semana funciona bárbaro el FA 1.8.8.... Hasta hoy anda en los dos móviles que tengo.. Uno con y otro sin root. Incluso el Control directo en la pantalla del auto funciona bien, que antes no dejaba, tenías que manejarlo desde pantalla del celular.... Ahora está muy bien todo.

**Comment by drviper1982** [comment]
can someone please tell me how can i run fermata auto on Android auto now???
please guys...my kids are killing me without it....

**Comment by drviper1982** [comment]
is there any official statment from google about this issue?

**Comment by AndreyPavlenko** [comment]
Try to install version 1.8.9 with KingInstaller

**Comment by Alejandro2454** [comment]
@andreypavlenko hola Andrey, instalé ayer con Kinginstaller la última
versión 1.8.10 y hasta el momento todo ok

**Comment by vldanch** [comment]
Does someone know when to expect a fix? 🤔

**Comment by AndreyPavlenko** [comment]
The latest version works with Android < 12. For Android 12 you need to downgrade AA to the version 6.9.

**Comment by Alfredo2190** [comment]
Para mi está funcionando muy bien hace ya un mes no he tenido problema.

**Comment by Alfredo2190** [comment]
Android 11

**Comment by Alejandro2454** [comment]
@Alfredo2190
Estimado, ¿tienes Android 12? ¿Eres root?
Saludos
Alejandro

**Comment by Alfredo2190** [comment]
No.. Android 11 y no soy root... Funciona muy bien  los Fermata Auto
1.8.10. Saludos

**Comment by vldanch** [comment]
@AndreyPavlenko On the Samsung Galaxy S21 phone, it rolled back to version AA 6.9.61372, but when connected via usb in the car, it is automatically updated via Google Play to the latest version 7.1.614574-release
What to do about this problem? 🤔

**Comment by AndreyPavlenko** [comment]
Have you tried to disable auto update in the goggle play settings?

**Comment by vldanch** [comment]
@AndreyPavlenko yes, i unchecked the auto update checkbox, but the update button appeared on the car screen, and it’s impossible to get around it...

![](https://i.imgur.com/AF7ovPe.jpg)

**Comment by fcaronte** [comment]
> @AndreyPavlenko yes, i unchecked the auto update checkbox, but the update button appeared on the car screen, and it’s impossible to get around it...
> 
> ![](https://camo.githubusercontent.com/c2bc589c9ac06fb6db3396c735e5dcc5890562308e07a4a7eb2ca7209e5e7a6d/68747470733a2f2f692e696d6775722e636f6d2f4146376f7650652e6a7067)

tested few minutes ago and on android 12 you need to use last 6.9 version "6.9.613744" other version won't work

**Comment by vldanch** [comment]
@fcaronte Where did you download this version?
for example from this [site](https://www.apkmirror.com/apk/google-inc/android-auto/android-auto-6-9-6137-release/android-auto-6-9-613744-release-android-apk-download/) does not work...
![](https://i.imgur.com/rA8sB8G.jpg)

**Comment by fcaronte** [comment]
> @fcaronte Where did you download this version? for example from this [site](https://www.apkmirror.com/apk/google-inc/android-auto/android-auto-6-9-6137-release/android-auto-6-9-613744-release-android-apk-download/) does not work... ![](https://camo.githubusercontent.com/c9097e9fc6f6c5b9c45386abb8a2fd35b7d24997e140ef8324d8e6bb34195067/68747470733a2f2f692e696d6775722e636f6d2f724138734238472e6a7067)

i'm using this version too from this site but before try to install you need to uninstall the new version and go to preinstalled version to be able to "downgrade"

**Comment by vldanch** [comment]
@fcaronte Wow, this option really works!
Thank you so much you are a kind person!
And no need to flash the phone, it's cool!!!

**Comment by abarth83** [comment]
downgrade AA v6.9.613744
install fermata auto
ok but reboot phone
 and fermata disappears :-(

**Comment by molinebis1** [comment]
After the downgrade to 6.9.xxx it worked. 
I disabled uodates in Google Play .
2 days later it updated, I do not how, to 7.1.614574
:-( 
I do not know how to avoid updates

**Comment by abarth83** [comment]
I have removed the shif on automatic update but in any case fermata 
 disappears when the phone is restarted

**Comment by fcaronte** [comment]
Keep attention because on andorid 12 the update is showed with a notification every time after you use android auto and so maybe you update with an unwanted click on this notification @molinebis1
